tt_metal/impl/buffers/buffer.hpp

```diff
@@ -289,7 +289,7 @@ BufferPageMapping generate_buffer_page_mapping(const Buffer &buffer);
 
 namespace detail {
 using PageAddress = uint32_t;
-using Deviceid = uint32_t;
+using Deviceid = umd::chip_id;
```

Comment:
I am a bit confused. Device vs Chip. 
What is the connection between `tt::tt_metal::Device` and `DeviceId`.
Is chip_id - like 0 in e75 and 0 and 1 in e150/n300

Reply:
chip id should be a logical ID representing a chip. n300 will have {0,1], t3000 will have {0,...,7}, tg {0,...,35}, and so on.

Reply:
Makes sense. DeviceId = ChipId is confusing because for me 

- gs/wh/bh are chips
- n300/n150/e150/e75 are devices
- t3000, galaxy are systems

I understand it is not a matter of this PR, but just want to flag that `using Deviceid = umd::chip_id;` is an absurd naming